### PR TITLE
Tournament lobby + dashboard

### PR DESCRIPTION
**Pull request by sgfost**
add tournament lobby, dashboard
add banner to the landing page
add tournament participation links to the navbar, landing page, and footer

### CODE REVIEW

**Comment by sgfost** [comment]
thanks for the matchmaking changes @alee. Dashboard is up and running, I'll put together an announcement banner + linkage tomorrow

File: client/src/plugins/ajax.ts [hunk 1]
```diff
@@ -55,7 +60,7 @@ export class AjaxRequest {
       ({ data, status }) => {
         if (status === 200) {
           this.store.commit("SET_USER", data.user);
-          if (data.user.isVerified) this.router.push({ name: LOBBY_PAGE });
+          if (data.user.isVerified) this.router.push({ name: FREE_PLAY_LOBBY_PAGE });
```
Comment (alee) [inline client/src/plugins/ajax.ts:64]: Should we add another check to see if we should funnel them to the tournament dashboard instead?

**Review by alee — APPROVED** [review]
great work as usual @sgfost !

File: client/src/plugins/ajax.ts (hunk 1 above)
Comment (sgfost) [reply client/src/plugins/ajax.ts:64]: Oh yeah I suppose that would be one way to fix it, though may be confusing to route to one or the other when both are active. I think routing to home is fine until there's a way to pass a next param to the login routes

**Pull request merged**